### PR TITLE
 [libmaxminddb] Updated version to 1.9.1

### PR DESCRIPTION
**Pull request by TXC**
- [X] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md)
- [X] SHA512s are updated for each updated download
- [X] The "supports" clause reflects platforms that may be fixed by this new version
- [X] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.
- [X] Any patches that are no longer applied are deleted from the port's directory.
- [X] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [X] Only one version is added to each modified port's versions file.

### CODE REVIEW

**Comment by TXC** [comment]
@microsoft-github-policy-service agree

File: ports/libmaxminddb/fix-lib-to-share.patch [hunk 1]
```diff
@@ -0,0 +1,13 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 5411be0..e92c495 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -97,7 +97,7 @@ install(TARGETS maxminddb
+ install(EXPORT maxminddb
+         FILE maxminddb-config.cmake
+         NAMESPACE maxminddb::
+-        DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/maxminddb)
++        DESTINATION ${CMAKE_INSTALL_DATADIR}/cmake/maxminddb)
```
Comment (dg0yt) [inline ports/libmaxminddb/fix-lib-to-share.patch:10]: Do we need the patch, or can we pass `CONFIG_PATH lib/cmake/maxminddb` to `vcpkg_cmake_config_fixup`?

File: ports/libmaxminddb/portfile.cmake [hunk 2]
```diff
@@ -3,22 +3,27 @@ vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
...
-        -DDISABLE_INSTALL_HEADERS=ON
+        -DCMAKE_DEBUG_POSTFIX=d
 )
-
 vcpkg_cmake_install()
 vcpkg_copy_pdbs()
 
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/share")
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
+vcpkg_fixup_pkgconfig()
```
Comment (dg0yt) [inline ports/libmaxminddb/portfile.cmake:26]:
```suggestion
vcpkg_fixup_pkgconfig()
vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/maxminddb PACKAGE_NAME maxminddb)
```
(Why do upstream repo names never match CMake package names?)

File: ports/libmaxminddb/fix-lib-to-share.patch (hunk 1 above)
Comment (TXC) [reply ports/libmaxminddb/fix-lib-to-share.patch]: I was unsure at first, therefore i implemented the patch.
But i tried your suggestion, and it worked.

Thanks!

File: ports/libmaxminddb/portfile.cmake (hunk 2 above)
Comment (TXC) [reply ports/libmaxminddb/portfile.cmake:23]: Also implemented the package name.

Blame GNU/Linux, it's their fault all libraries have the "lib"-prefix, and some developers think that "i'm coding a library, i must prefix it with lib" 😆

**Pull request merged**